src/java/org/apache/cassandra/config/Config.java

```diff
@@ -525,17 +602,17 @@
     /** This feature allows denying access to operations on certain key partitions, intended for use by operators to
      * provide another tool to manage cluster health vs application access. See CASSANDRA-12106 and CEP-13 for more details.
      */
-    public volatile boolean enable_partition_denylist = false;
```

Comment:
These are new in 4.1, so no issue w/ the rename?

Reply:
Yes, anything that is not released even in alpha can be still changed without a need of backward compatibility. There will be more after I do a rebase. Trunk is alive.